server/src/main/java/org/jboss/as/server/ServerEnvironmentService.java

```diff
@@ -22,13 +22,16 @@
 
 package org.jboss.as.server;
 
+
 import org.jboss.msc.service.Service;
 import org.jboss.msc.service.ServiceName;
 import org.jboss.msc.service.ServiceTarget;
 import org.jboss.msc.service.StartContext;
 import org.jboss.msc.service.StartException;
 import org.jboss.msc.service.StopContext;
 
+import static org.jboss.as.server.ServerService.SERVER_ENVIRONMENT_CAPABILITY;
+
```

Comment:
Sorry @parsharma , I missed this one, static imports go before, take a look at your IDE settings and ensure you are using the Eclipse Formatter with the WildFly profile. The ide configs can be found here herehttps://github.com/wildfly/wildfly-core/tree/main/ide-configs

Reply:
Done, thanks